### DISCUSSION TITLE
Is there the chance to force the position of a tooltip?

### DISCUSSION THREAD

**Discussion by LauGroupie95** [buildo/bento-design-system#367] [Q&A] [answered]
Hi @veej and @gabro, 
as you suggested us, we should use a tooltip for an IconButton and it should be on top of the IconButton (see 1st image), however, the actual implementation shows the tooltip on the right (see 2nd image).
Is there the chance to force the position of a tooltip?

What I want to achieve:
![image](https://user-images.githubusercontent.com/63547770/187628579-6f1d53ee-2e8a-4c7e-a559-9bf4eda99ab3.png)

What I've obtained:
![image](https://user-images.githubusercontent.com/63547770/187628734-fbafcdc0-b458-4302-95c5-fdef9ce38eaa.png)

Thank you in advance!

**Comment by veej** [comment, marked as answer]
Hi @LauGroupie95.
At the moment, the position of the Tooltip is computed automatically, but it could make sense to force a different position.
I'll open a task to track the development of the new feature.

**Reply by gabro** [reply]
Hi @LauGroupie95, we've released this feature as part of v0.15.2

**Reply by LauGroupie95** [reply]
Great, thanks! I'll try in a while
